src/Bundle/DependencyInjection/SyliusResourceExtension.php

```diff
@@ -135,4 +136,68 @@ private function loadResources(array $loadedResources, ContainerBuilder $contain
             }
         }
     }
+
+    private function createTranslationParameters(array $config, ContainerBuilder $container): void
+    {
+        $this->createEnabledLocalesParameter($config, $container);
+        $this->createDefaultLocaleParameter($config, $container);
+    }
+
+    private function createEnabledLocalesParameter(array $config, ContainerBuilder $container): void
+    {
+        $enabledLocales = $config['translation']['enabled_locales'];
+
+        if (count($enabledLocales) > 0) {
+            $container->setParameter('sylius.resource.translation.enabled_locales', $enabledLocales);
+
+            return;
+        }
+
+        if ($container->hasParameter('locale')) {
+            $container->setParameter('sylius.resource.translation.enabled_locales', [$container->getParameter('locale')]);
+
+            return;
+        }
+
+        if ($container->hasParameter('kernel.enabled_locales')) {
+            $kernelEnabledLocales = (array) $container->getParameter('kernel.enabled_locales');
+
+            if (count($kernelEnabledLocales) > 0) {
+                $container->setParameter('sylius.resource.translation.enabled_locales', $container->getParameter('kernel.enabled_locales'));
+
+                return;
+            }
+        }
+
+        $container->setParameter('sylius.resource.translation.enabled_locales', ['en']);
+    }
+
+    private function createDefaultLocaleParameter(array $config, ContainerBuilder $container): void
+    {
+        $defaultLocale = $config['translation']['default_locale'];
+
+        if (is_string($defaultLocale)) {
+            $container->setParameter('sylius.resource.translation.default_locale', $defaultLocale);
+
+            return;
+        }
+
+        if ($container->hasParameter('locale')) {
+            $container->setParameter('sylius.resource.translation.default_locale', $container->getParameter('locale'));
+
```

Comment:
Here's where a deprecation notice should be triggered warning users that this parameter will no longer used in 2.0 and they should set the `default_locale` option though configuration.